src/parametersFactory.js

```diff
@@ -0,0 +1,16 @@
+import parameterFactory from './parameterFactory';
+import parametersDuplicatesFilter from './parametersDuplicatesFilter';
+
+export function filterOutDuplicatedVariablesEndingWithBrackets(array) {
+  return parametersDuplicatesFilter(array, 'variable', '[]');
```

Comment:
Added to filter out duplicated filters.

Reply:
I've dropped this commit, IMO your solution is overkill. What i had in mind was just a regex : https://github.com/api-platform/admin/blob/e81e8537b9fd2338458f107183e87c4dae71821e/src/parameterFactory.js#L63

Maybe you discuss about this with @dunglas and there is a reason i am missing. Anyway, i saved your work on another branch, don't worry : https://github.com/jfthuillier/admin/commits/filters-backup.

Reply:
It was to keep properties in the case where the API returns a variable ending with square brackets and not the same variable without square brackets. For exemple, to keep intact :
`{ variable: "foo[]" }`
But I am not sure if this case is possible with API Platform. So it might be overkill indeed. I will discuss this with @dunglas